server/handlers/cdn-caching.js

```diff
@@ -34,23 +32,26 @@ exports.addCacheHeadersToResult = function addCacheHeadersToResult({
           `block-all-mixed-content;`
       );
     } else {
-      res.setHeader(
-        "Cache-Control",
-        `public,max-age=15,s-maxage=${sMaxAge},stale-while-revalidate=1000,stale-if-error=14400`
-      );
-      cdnProviderVal === "akamai" &&
+      if (networkOnly) {
+        res.setHeader("Cache-Control", `public,s-maxage=${sMaxAge}`);
+        res.setHeader("Cloudflare-CDN-Cache-Control", `stale-if-error=14400`);
```

Comment:
A concern is that without `stale-while-revalidate` on cloudflare, frontend will take a lot of time to render as route-data.json takes time to compute on the server side. Is it possible to add that directive as well?

Reply:
in that case, use won't get fresh data on first load.